integration/js/app/video_test/video_test.ts

```diff
@@ -0,0 +1,199 @@
+import {
+  ConsoleLogger,
+  DefaultDeviceController,
+  MeetingSessionConfiguration,
+  DefaultMeetingSession,
+  LogLevel,
+  AudioVideoObserver,
+  MeetingSessionStatus,
+  VideoTileState,
+  AudioVideoFacade,
+  MeetingSession,
+  Versioning,
+  MeetingSessionStatusCode
+} from '../../../../src/index';
+
+  export class DemoMeetingApp implements AudioVideoObserver {
```

Comment:
nit: Remove 2-space indentation from this file

Reply:
Done, resubmitted.